spec/datadog/tracing/integration_spec.rb

```diff
@@ -548,11 +548,13 @@ def sample!(trace)
             span.service = 'my.service'
           end
 
-          sleep(1)
+          write.write('!') # Signals that this fork is ready
+          sleep(5)
```

Comment:
If I understand it correctly, the intention of this spec is to check that when the process receives a SIGTERM, the ddtrace shutdown runs as expected.

So, like in another PR I just reviewed, this `sleep` is always supposed to be interrupted, it's never supposed to "run out" -- so perhaps it should be a `sleep` without a timeout?

Reply:
I changed this so that it exists "badly" if the sleep is not interrupted. This will ensure the test fails if not interrupted.

I don't like the idea of a non-timeout wait in our test suite, given it can always go wrong and hand the test suite.